server/vcr-server/vcr_server/custom_settings_bcgov.py

```diff
@@ -45,11 +45,82 @@ def list_related_to_relations(self, request, pk=None):
     from api.v2.models.TopicRelationship import TopicRelationship
     from rest_framework.response import Response
 
-    parent_queryset = TopicRelationship.objects.filter(topic=pk).all()
-    serializer = CustomTopicRelationshipSerializer(
-        parent_queryset, many=True, relationship_type="to"
-    )
-    return Response(serializer.data)
+    topic_relationships = TopicRelationship.objects.filter(topic=pk).all()
+
+    data = [
```

Comment:
In a couple spots I query and serialize manually. This is a ~5x speed increase over DRF. And much less chatty with the db so it should improve overall performance as well.